extensions/cl_ext_float_atomics.asciidoc

```diff
@@ -0,0 +1,734 @@
+
+// In addition to the requirements described in the OpenCL C 3.0 specification,
+// requires the __opencl_c_ext_fp16_global_atomic_add feature
+// or the __opencl_c_ext_fp16_local_atomic_add feature (for atomic_half),
+// requires the __opencl_c_ext_fp32_global_atomic_add feature
+// or the __opencl_c_ext_fp32_local_atomic_add feature (for atomic_float), or
+// requires the __opencl_c_ext_fp64_global_atomic_add feature
+// or the __opencl_c_ext_fp64_local_atomic_add feature (for atomic_double).
+C atomic_fetch_add(volatile A *object, M operand)
+C atomic_fetch_sub(volatile A *object, M operand)
+C atomic_fetch_add_explicit(volatile A *object, M operand, memory_order order)
+C atomic_fetch_sub_explicit(volatile A *object, M operand, memory_order order)
+C atomic_fetch_add_explicit(volatile A *object, M operand,
+    memory_order order, memory_scope scope)
+C atomic_fetch_sub_explicit(volatile A *object, M operand,
+    memory_order order, memory_scope scope)
+
+// atomic_fetch_min / atomic_fetch_max:
+
+// In addition to the requirements described in the OpenCL C 3.0 specification,
+// requires the __opencl_c_ext_fp16_global_atomic_min_max feature (for atomic_half),
+// requires the __opencl_c_ext_fp32_global_atomic_min_max feature (for atomic_float), or
+// requires the __opencl_c_ext_fp64_global_atomic_min_max feature (for atomic_double).
+C atomic_fetch_min(volatile __global A *object, M operand)
+C atomic_fetch_max(volatile __global A *object, M operand)
+C atomic_fetch_min_explicit(volatile __global A *object, M operand, memory_order order)
+C atomic_fetch_max_explicit(volatile __global A *object, M operand, memory_order order)
+C atomic_fetch_min_explicit(volatile __global A *object, M operand,
+    memory_order order, memory_scope scope)
+C atomic_fetch_max_explicit(volatile __global A *object, M operand,
+    memory_order order, memory_scope scope)
+
+// In addition to the requirements described in the OpenCL C 3.0 specification,
+// requires the __opencl_c_ext_fp16_local_atomic_min_max feature (for atomic_half),
+// requires the __opencl_c_ext_fp32_local_atomic_min_max feature (for atomic_float), or
+// requires the __opencl_c_ext_fp64_local_atomic_min_max feature (for atomic_double).
+C atomic_fetch_min(volatile __local A *object, M operand)
+C atomic_fetch_max(volatile __local A *object, M operand)
+C atomic_fetch_min_explicit(volatile __local A *object, M operand, memory_order order)
+C atomic_fetch_max_explicit(volatile __local A *object, M operand, memory_order order)
+C atomic_fetch_min_explicit(volatile __local A *object, M operand,
+    memory_order order, memory_scope scope)
+C atomic_fetch_max_explicit(volatile __local A *object, M operand,
+    memory_order order, memory_scope scope)
+
+// In addition to the requirements described in the OpenCL C 3.0 specification,
+// requires the __opencl_c_ext_fp16_global_atomic_min_max feature
+// or the __opencl_c_ext_fp16_local_atomic_min_mas feature (for atomic_half),
```

Comment:
```suggestion
// or the __opencl_c_ext_fp16_local_atomic_min_max feature (for atomic_half),
```

This `_mas` -> `_max` typo occurs in 8 more places further down.

Reply:
Thank you, I think I've fixed all of these (my grep for `mas` didn't find anything).

I didn't rev the document version since this is just a typo fix.